string angleOne
string angleTwo
string angleThree
string angleFour